cmake/DD4hepBuild.cmake

```diff
@@ -630,7 +630,7 @@ function(dd4hep_add_dictionary dictionary )
     )
 
   #  Install the binary to the destination directory
-  install(FILES ${output_dir}/${dictionary}_rdict.pcm DESTINATION lib)
+  install(FILES ${output_dir}/${dictionary}_rdict.pcm DESTINATION ${CMAKE_INSTALL_LIBDIR})
```

Comment:
Probably just check that `CMAKE_INSTALL_LIBDIR` is set otherwise set it to `lib`